svg.js.d.ts

```diff
@@ -548,13 +548,12 @@ declare module "@svgdotjs/svg.js" {
         to(a: any): Morphable;
     }
 
+    type ListEachCallback<T> = (el: T, index: number, list: T[]) => any

```

Comment:
Just saw this: The third paramater `list` will be a List, too. So I think it has to be `list: List<T>`